xml/Microsoft.Build.Evaluation/GlobResult.xml

```diff
@@ -139,7 +140,9 @@
       </ReturnValue>
       <Docs>
         <summary>
-            A <see cref="T:Microsoft.Build.Globbing.IMSBuildGlob" /> representing the include globs. It also takes the excludes and relevant removes into consideration.
+            A <see cref="T:Microsoft.Build.Globbing.IMSBuildGlob" /> representing the globs found in an item `Include` attribute.
+            It also takes into consideration the strings from `Exclude` attributes that are present on that item, and strings from
```

Comment:
I'm unclear what "takes into consideration" means. It's also not clear what "it" refers to.